src/lib/watcher/runner.ts

```diff
@@ -126,25 +119,51 @@ register({
     IPC.client.senders.moduleImported({ filePath })
   })
 
-  if (!process.env.NEXUS_EVAL) {
-    throw new Error('process.env.NEXUS_EVAL is required')
-  }
+  const startModuleFileName = layout.sourceRoot + '/index.ts'
+  const startModule = tsNodeRegister.compile(
```

Comment:
Would be nice to have a consistent TS program system in Nexus. Having ts-node around doesn't give us much and it increases internal complexity by having to work with two systems.

Reply:
The way I see it, `ts-node` should only be used for it's "on-the-fly" ts compilation. If you want to be consistent here, I'd suggest to use our internal `transpileModule` function in `tsc.ts`